test/double_check_test.dart

```diff
@@ -21,16 +21,11 @@ void main() {
 
     synchronize.sources.forEach((sourcePath, targetPath) {
       test(targetPath, () {
-        var message = "$targetPath is out-of-date.\n"
-            "Run pub run grinder to update it.";
-
-        var target = File(targetPath).readAsStringSync();
-        var match = checksumPattern.firstMatch(target);
-        if (match == null) fail(message);
-
-        var source = File(sourcePath).readAsBytesSync();
-        var expectedHash = sha1.convert(source).toString();
-        expect(match[1], equals(expectedHash), reason: message);
+        if (File(targetPath).readAsStringSync() !=
+            synchronize.synchronizeFile(sourcePath)) {
+          fail("$targetPath is out-of-date.\n"
+              "Run `pub run grinder` to update it.");
```

Comment:
The standalone `pub` command no longer exists. This should be `dart run grinder`.